include/libopencm3/stm32/sdio.h

```diff
@@ -79,13 +79,14 @@
 /* --- SDIO_POWER values --------------------------------------------------- */
 
 #define SDIO_POWER_PWRCTRL_SHIFT	0
+#define SDIO_POWER_PWRCTRL_MASK		(0x3 << SDIO_POWER_PWRCTRL_SHIFT)
```

Comment:
For consistency with _most_ other places, when both SHIFT and MASK are defined, the mask is unshifted.